test/test_base58.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 John Newbery
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test base58.py"""
+import unittest
+
+import base58
+from binascii import unhexlify
+
+TEST_CASES = [
```

Comment:
Great. I see that you've taken these from the bitcoind repo. Can you add a comment saying something like "test vectors from https://github.com/bitcoin/bitcoin/blob/b5e4b9b5100ec15217d43edb5f4149439f4b20a5/src/test/data/base58_encode_decode.json"

Reply:
Sure thing!